security/protecting-personal-or-identifying-information.md

```diff
@@ -0,0 +1,17 @@
+# Protecting Personal or Identifying Information
+
+Data privacy and security should be made a priority when developing software in
+an effort to protect personal or identifying information.
+
+Some individuals, due to their backgrounds or circumstances, might be at a
+higher risk of harmful consequences from privacy violations.
+
+Examples include:
+
+- Survivors of domestic abuse or those who are trying to escape domestic abuse
+- LGBTQIA+ individuals, especially if their housing is dependent on someone who
+  might evict them upon learning this, or if they're in a number of other unsafe
+circumstances
```

Comment:
```suggestion
- LGBTQIA+ individuals who might be in an unsafe circumstance in relation to their identity, including if their housing is dependent on someone who might evict them because of it
```